packages/core/react/use-assistant.ts

```diff
@@ -15,39 +15,26 @@ export function experimental_useAssistant({
   threadId?: string | undefined;
 }) {
   const [messages, setMessages] = useState<Message[]>([]);
-  const [input, setInput] = useState('');
   const [threadId, setThreadId] = useState<string | undefined>(undefined);
   const [status, setStatus] = useState<AssistantStatus>('awaiting_message');
   const [error, setError] = useState<unknown | undefined>(undefined);
 
-  const handleInputChange = (e: any) => {
-    setInput(e.target.value);
-  };
-
-  const submitMessage = async (e: any) => {
+  // from form submit:
+  const submitMessage = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
 
-    if (input === '') {
-      return;
-    }
+    const formData = new FormData(e.currentTarget);
```

Comment:
I used a formData instead because I find it odd to use a form on the client side and send a json, but it's not a big deal.